File: 2630-alternating-digit-sum/alternating-digit-sum.py
class Solution:
    def alternateDigitSum(self, n: int) -> int:
        c=0
        n=str(n)
        for i in range(len(n)):
            if i%2==0:
                c+=int(n[i])
            else:
                c+=int(n[i])*(-1)
        return c

File: 2630-alternating-digit-sum/alternating-digit-sum.cs
public class Solution {
    public int AlternateDigitSum(int n) {
        int c=0;
        string s=n.ToString();
        for(int i=0; i<s.Length; i++){
            c+= (i%2==0) ? s[i]-'0' : -(s[i]-'0');
        }
        return c;
    }
}